.travis.yml

```diff
@@ -0,0 +1,6 @@
+language: node_js
+node_js: 
+  - "stable"
+cache:
+  directories:
+    - "node_modules"
```

Comment:
@edisonljh I'm not sure if this is the correct folder to cache. I think it should be either `$HOME/.yarn-cache` or `$HOME/.npm` depending on which package manager this repository is using.